stake-pool/py/pyproject.toml

```
@@ -0,0 +1,67 @@
+[project]
+name = 'stake-pool'
+version = '1.0'
+authors = [
+  { name= 'joncinque'},
```

Can this be the solana foundation? ie
```suggestion
  { name="Solana Maintainers", email="maintainers@solana.foundation" },
```
Actually, we use that address everywhere, and I'm not sure if it's even valid :laughing: could you check to see that it's still good?